src/Stack/Assets.php

```diff
@@ -35,6 +35,7 @@
  * @license   https://github.com/contentstack/contentstack-php/blob/master/LICENSE.txt MIT Licence
  * @link      https://pear.php.net/package/contentstack
  * */
+#[\AllowDynamicProperties]
```

Comment:
Is `#[\AllowDynamicProperties]` a deliberate part of the code or something that was written unintentionally?